services/IoT/applications/libraries/python.md

```diff
@@ -447,15 +443,15 @@ organization. A request must be no larger than 512KB.
 ## Device type operations
 {: #device_type_ops}
 
-The device types that you create in your organization can be used to create templates for adding devices. Through the {{site.data.keyword.iot_short_notm}} API features, your applications can list, create, delete, view, or update device types in your organization.
+The device types that you create in your organization can be used to create templates for adding devices. By using the {{site.data.keyword.iot_short_notm}} API features, your applications can list, create, delete, view, or update device types in your organization.
 
 For information about query parameters, the request & response model, and HTTP status codes, see the 'Device types' section of the [IBM {{site.data.keyword.iot_short_notm}} API](https://docs.internetofthings.ibmcloud.com/swagger/v0002.html) documentation.
 
 
 ### Retrieving all device types
 
-You can use the ``getAllDeviceTypes()`` method to retrieve all of the device types that are in your {{site.data.keyword.iot_short_notm}} organization.
-The response contains parameters required by the application to retrieve the dictionary *results* from the response to get the array of devices returned. Other parameters in the response are required to make further calls, for example, the ``_bookmark`` element can be used to page through results. Issue the first request without specifying a bookmark, then take the bookmark returned in the response and provide it on the request for the next page. Repeat this process until the end of the result set indicated by the absence of a bookmark. Each request must use exactly the same values for the other parameters, or the results will be undefined.
+You can use the ``getAllDeviceTypes()`` method to retrieve all the device types that are in your {{site.data.keyword.iot_short_notm}} organization.
+The response contains parameters that are required by the application to retrieve the dictionary *results* from the response to get the array of devices returned. Other parameters in the response are required to make more calls, for example, the ``_bookmark`` element can be used to page through results. Issue the first request without specifying a bookmark, then take the bookmark that is returned in the response and provide it on the request for the next page. Repeat this process until the end of the result set, which is indicated by the absence of a bookmark. Each request must use exactly the same values for the other parameters, or the results will be undefined.
```

Comment:
"to get the array of devices returned"
does this mean "to return the array of devices" or "to get the array of devices that is returned"